inspectit-oce-documentation/src/docs/asciidoc/_includes/configuration/configuration-basics.adoc

```diff
@@ -0,0 +1,69 @@
+=== Configuration Basics
+
+inspectIT OCE tries to implement the zero-configuration approach but lets you externalize the configuration and influence every bit of its configuration if this is required.
+Internally it uses the Spring-based `PropertySource` order to allow overriding of configuration values.
+Configuration properties are considered in inspectIT OCE in the following order of priority:
```

Comment:
"in the following order, where properties specified first overwrite properties found afterwards " or some other way of clarifying what actually is the overwriting order?